packages/design-picker/src/components/theme-card/style.scss

```diff
@@ -50,9 +50,12 @@ $theme-card-info-margin-top: 16px;
 	position: relative;
 }
 
+.theme-card__image-wrapper {
+	box-shadow: 0 15px 25px rgba(0, 0, 0, 0.05);
+}
+
 .theme-card__image-container {
 	border: 1px solid rgba(0, 0, 0, 0.1);
-	filter: drop-shadow(0 15px 25px rgba(0, 0, 0, 0.05));
```

Comment:
Question: Why not just replace it with `box-shadow`?

Reply:
Hmmm when I tried before, the `box-shadow` it was truncated, but it doesn't seem to be the case anymore. Good catch!